changelog/18.0/18.0.0/summary.md

```diff
@@ -63,6 +64,8 @@ for a response from all the tablets. Originally `EmergencyReparentShard` was mea
 We have realized now that there are cases when the replication is broken but all the tablets are reachable. In these cases, it is advisable to 
 call `EmergencyReparentShard` with `--wait-for-all-tablets` so that it doesn't ignore one of the tablets.
 
+#### <a id="new-flag-toggle-ers"/>VTOrc flag `--allow-emergency-reparent`
+
```

Comment:
These 2 lines should go. They duplicate lines 45-46.

Reply:
Whoops, yes